src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -281,6 +281,52 @@ Below is a list of available validate commands:
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
+### `gather`
+
+The `subctl gather` command is a tool that collects various information from clusters to aid in troubleshooting a
+Submariner deployment, including Kubernetes resources and Pod logs. Clusters from which information is gathered are provided
+via `the --kubeconfig` flag (or the `KUBECONFIG` environment variable). By default it will gather information from all the cluster contexts contained
```

Comment:
```suggestion
via the `--kubeconfig` flag (or the `KUBECONFIG` environment variable). By default it will gather information from all the cluster contexts contained
```